interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -39,21 +39,22 @@ function postcalendar_admin_modifyconfig($msg = '', $showMenu = true)
 
     $output->SetInputMode(_PNH_VERBATIMINPUT);
 
-    $header = "<html><head><title>" . xlt("Calendar") . "</title></head>";
+    $header = "<html><head><title>" . xlt("Calendar") . "</title>";
+    $header .= '<link rel="stylesheet" href="../../../public/assets/bootstrap/dist/css/bootstrap.min.css"></head>';
```

Comment:
Rather than include bootstrap directly do this, would consider:
1. `use OpenEMR\Core\Header;` at line 30
2. change above line to:
```
$header .= Header::setupHeader('', false)  . '</head>';
```